apps/src/templates/checkbox-dropdown.module.scss

```diff
@@ -0,0 +1,4 @@
+.affectAllButton {
+  display: flex;
+  justify-content: center;
+}
```

Comment:
This is just placeholder styling for the 'Select all' and 'Clear all' buttons for each filter that Denys will be styling.